packages/components/menu/src/menu-item.vue

```diff
@@ -83,14 +83,14 @@ export default defineComponent({
       active,
     })
 
-    const handleClick = () => {
+    const handleClick = (e) => {
       if (!props.disabled) {
         rootMenu.handleMenuItemClick({
           index: props.index,
           indexPath: indexPath.value,
           route: props.route,
         })
-        emit('click', item)
+        emit('click', item, e)
```

Comment:
Thanks for your contribution, I feel like it would make more sense to put the event first, since then the modifiers on the event will also take effect, but that would be a breaking change.